CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 1.12.0 - 2023-09-27
+
+* Potentially breaking changes
+  * `:nerves >= 1.8.0` is now required to use this version of `nerves_bootstrap`.
+    If you keep up-to-date, then this won't be an issue for you.
+
+* Updates
+  * Removed legacy tasks which are now maintained in [`:nerves`](https://github.com/nerves-project/nerves)
```

Comment:
not

Reply:
I don't really understand what you're saying here?

Reply:
Sorry, was mobile. I tried to change `now` to `not` since the legacy tasks a no longer maintained. It may be better to just say that the legacy tasks are removed and leave it at that.

Reply:
Changed. How about this?